common/features/config/networks/static/reducer.ts

```diff
@@ -698,6 +699,58 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       max: 60,
       initial: 20
     }
+  },
+
+  ETI: {
+    id: 'ETI',
+    name: 'ETI',
+    unit: 'ETI',
+    chainId: 101,

```

Comment:
@suryanshkumr  Does ETI support EIP-155?

Regardless of whether EIP-155 is actually supported or not, chainId 101 is already in use by WebChain Network.

It is not safe for both networks to use the same chainId - I'd suggest a hardfork ASAP to pick a chainId that is unique to EtherInc